base/sparse/higherorderfns.jl

```diff
@@ -63,8 +63,12 @@ end
 
 # (2) map[!] entry points
 map{Tf}(f::Tf, A::SparseVecOrMat) = _noshapecheck_map(f, A)
+map{Tf,N}(f::Tf, A::SparseMatrixCSC, Bs::Vararg{SparseMatrixCSC,N}) =
```

Comment:
This is the intersection of the definition on the next line and the one on line 937 (for `SparseOrStructuredMatrix`).